jekyll/_cci2/local-cli.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: classic-docs
+title: "The CircleCI Local Command Line Interface (CLI)"
```

Comment:
Let's make this the same as the short title 'Using the Local CircleCI CLI'

Reply:
No problem. Pushing update shortly.